Corona-Warn-App/src/main/res/menu/menu_main.xml

```diff
@@ -1,5 +1,5 @@
-<menu xmlns:tools="http://schemas.android.com/tools"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Unrelated changes?

Reply:
was touched during commit history. I think its ok to mob up sometimes. Its clean reformat.